rae_hw/launch/control.launch.py

```diff
@@ -43,24 +43,24 @@ def launch_setup(context, *args, **kwargs):
                 'rae_hw'), 'config', 'ekf.yaml')],
     )
 
-    imu_comp_filt = Node(
-        package='imu_complementary_filter',
-        executable='complementary_filter_node',
-        name='complementary_filter_gain_node',
-        namespace=LaunchConfiguration('namespace'),
-        output='screen',
-        parameters=[
-            {'do_bias_estimation': True},
-            {'do_adaptive_gain': True},
-            {'orientation_stddev': 0.001},
-            {'use_mag': False},
-            {'gain_acc': 0.04},
-            {'gain_mag': 0.01},
-        ],
-        remappings=[
-            ('imu/data_raw', '/rae/imu/data'),
-        ]
-    )
+    # imu_comp_filt = Node(
```

Comment:
Why is it commented out?

Reply:
I do not use anymore complementary filter to obtain orientation(I used on rev4 because it have BMI IMU, that do not support GAME_ORIENTATION_VECTOR), so with BNO on rev5 we got directly the orientation.

Reply:
If it's no longer used then it can be deleted instead of commented out to keep the launch file clean